wp_localization/README.md

```diff
@@ -22,17 +22,17 @@ The localization process runs automatically every night via a Buildkite pipeline
 1. **Generates the source en-US PO file** from `localization/en-US/main.ftl` and commits it to `glotpress/en-US.pot`
 2. **Downloads latest translations** from GlotPress for all supported locales as PO files
 3. **Converts downloaded PO files to Fluent format** and updates the corresponding `localization/*/main.ftl` files
-4. **Commits the updated translation files** to the repository
+4. **Creates a Pull Request** with the updated translation files and the updated original en-US PO file in the repository
 
 This means that translation updates happen automatically without developer intervention.
```

Comment:
```suggestion
Once you merge the Pull Request into `trunk`, the update `glotpress/en-US.pot` file will be picked up by a wpcom job to import it into GlotPress, so that translators can start working on the new strings.

This means that translation updates happen automatically without developer intervention.
```